packages/material-ui-lab/src/ClockPicker/ClockPointer.tsx

```diff
@@ -67,12 +100,15 @@ class ClockPointer extends React.Component<ClockPointerProps> {
 
   state = {
     toAnimateTransform: false,
-    // eslint-disable-next-line react/no-unused-state
     previousType: undefined,
   };
 
   render() {
-    const { classes, hasSelected, isInner, type, value, ...other } = this.props;
+    // TODO migrate to hook and use useThemeProps.
```

Comment:
Looks like it is internal so we may even don't need the theme props.

Reply:
agreed